### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Default Screen_Keyboard: Unable to access synced and manage buttons using keyboard

### ISSUE THREAD

**Issue by Ashwinipola26** [microsoft/vscode-azurefunctions#2685]
### GitHubTags:
#Azure Tools for VS Code;#A11ySev1;#A11yTCS;#BM-Azure Tools for VS Code-MAC-Feb2021;#DesktopApp;#WCAG2.1.1;#MAC; #Keyboard;#AINotInScope

### OS Details:
macOS Catalina Version 10.15.7

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z(6 days ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin X64 19.6.0
Azure Function Version: 1.2.1

### Note:
Able to access when VoiceOver is on with Control+Option right arrow , but unable to access when VoiceOver is off using Keyboard

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab till Synced and Manage button and check whether able to access only using keyboard without VoiceOver.

### Actual Result:
Unable to access synced and manage buttons using keyboard

### Expected Result:
User should be able to access synced and manage buttons using keyboard and also using VoiceOver.

### User Impact:
Keyboard only users will not be able to access Synced and Manage buttons

### Attachment for Reference:

[Uploading Bug_2685_Unable to access Synced and Manage button using keyboard.zip…]()
